pylintrc

```diff
@@ -78,67 +72,6 @@ disable=print-statement,
         useless-suppression,
         deprecated-pragma,
         use-symbolic-message-instead,
-        apply-builtin,
```

Comment:
@naik-aakash any reason why we have to remove all of them or is this an issue with not having the latest main branch merged?

Reply:
Ah I removed them as they seem deprecated and made the lint workflow print a whole lot of warnings,